container-engine-lib/lib/backend_impls/docker/object_attributes_provider/label_value_consts/label_value_consts.go

```diff
@@ -11,10 +11,11 @@ const (
 	//
 	//   If you add new immutable values to this section, MAKE SURE TO UPDATE THE UNIT TEST!
 	//
-	appIdLabelValueStr                      = "kurtosis"
-	engineContainerTypeLabelValueStr        = "kurtosis-engine"
-	logsDatabaseContainerTypeLabelValueStr  = "kurtosis-logs-db"
-	logsCollectorContainerTypeLabelValueStr = "kurtosis-logs-collector"
+	appIdLabelValueStr                       = "kurtosis"
+	engineContainerTypeLabelValueStr         = "kurtosis-engine"
+	logsDatabaseContainerTypeLabelValueStr   = "kurtosis-logs-db"
```

Comment:
Do we still have a log DB? What is this used for?

Reply:
removing all log DB/loki elements